content/en/kb/sector-removal/index.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Troubleshooting Sector Removal"
+description: "This is a solution issues with sector removals and terminations."
+date: 2022-04-04T12:00:35+01:00
+lastmod: 2022-04-04T12:00:35+01:00
+draft: true
+menu:
+  kb:
+    parent: "browse"
+toc: false
+pinned: false
+types: ["solution"]
+areas: ["OS", "Linux", "Lotus Node"]
+---
+
+# Problem:
```

Comment:
Don't end a title with a colon. Freaks out URLs sometimes. Also, any chance we can make these titles `h2` tags instead of `h1` tags? There should only be one `h1` on the page, and that's specified by the `title: "Troubleshooting Sector Removal"` line.